README.md

```diff
@@ -1 +1,161 @@
-# node-modules-path
\ No newline at end of file
+# Paths, Folders and Modules
+
+## Overview
+
+This lesson will cover how to work with paths, use them when importing modules. We'll also show a code organization technique where you can import folders.
+
+## Objectives
+
```

Comment:
Think about objectives from "Students will be able to..." perspective. objectives should be tangible can dos